src/core/annotation.js

```diff
@@ -243,15 +243,16 @@ class Annotation {
   }
 
   /**
-   * Set the creation date.
+   * Set the contents.
    *
    * @public
    * @memberof Annotation
-   * @param {string} creationDate - PDF date string that indicates when the
-   *                                annotation was originally created
+   * @param {string} contents - Text to display for the annotation or, if the
+   *                            type of annotation does not display text, a
+   *                            description of the annotation's contents
    */
-  setCreationDate(creationDate) {
```

Comment:
I suppose that you could've moved this method into the `MarkupAnnotation` class instead, rather than duplicating the code below, but it probably doesn't matter too much either way so...

Reply:
+1 to that, I will probably need that piece of code too … so more repetitions coming :)

Reply:
I agree that that's actually better. Thanks, I'm going to fix that.

Reply:
Actually, looking at this once more you probably want to just leave it on the base `Annotation` class, since you're calling it from both the `MarkupAnnotation` and `PopupAnnotation` class!?

Reply:
Nice catch :) … since only modification date is now shown in UI popup (I think), I can probably drop it from PopupAnnotation, because it should not be there anyway (as you correctly pointed out - popup is not markup). I added it there in IRT PR, because it was added in base Annotation class and it needed to be changed to correct value in Parent->RT=Group situation. So I think, it is ok to move it to MarkupAnnotation and remove it from PopupAnnotation, because it should not be needed there.

Reply:
Done in the new commit. Thank you both!